lib/_http_client.js

```diff
@@ -133,10 +133,12 @@ function ClientRequest(options, cb) {
   if (self.socketPath) {
     self._last = true;
     self.shouldKeepAlive = false;
-    const optionsPath = {
-      path: self.socketPath
+    const createConnectionOptions = {
```

Comment:
It used to be `optionsPath` since the only option was the path. With this change we introduce non-path related options in there, which is why I felt renaming would make sense and I've chosen `createConnectionOptions` since we pass them to the `createConnection` function.